docs/gethue/content/en/posts/2022-10-11-create-iceberg-tables-in-just-2-steps.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Create Iceberg tables in Just 2 steps
+author: Hue Team
+type: post
+date: 2022-10-11T00:00:00+00:00
+url: /blog/2022-10-11-create-iceberg-tables-in-just-2-steps
+sf_thumbnail_type:
+  - none
+sf_thumbnail_link_type:
+  - link_to_post
+sf_detail_type:
+  - none
+sf_page_title:
+  - 1
+sf_page_title_style:
+  - standard
+sf_no_breadcrumbs:
+  - 1
+sf_page_title_bg:
+  - none
+sf_page_title_text_style:
+  - light
+sf_background_image_size:
+  - cover
+sf_social_sharing:
+  - 1
+sf_related_articles:
+  - 1
+sf_sidebar_config:
+  - left-sidebar
+sf_left_sidebar:
+  - Sidebar-2
+sf_right_sidebar:
+  - Sidebar-1
+sf_caption_position:
+  - caption-right
+sf_remove_promo_bar:
+  - 1
+ampforwp-amp-on-off:
+  - default
+categories:
+  - Version 4.11
+  - Query
+
+---
+
+As all of you might know, Hue started supporting many dialects like Hive, Impala, Phoenix, SparkSQL etc. In the recent past 1 year we have added many feature or improvments for eg [Local file importer](/blog/2021-05-26-improved-hue-importer-select-a-file-choose-a-dialect-create-a-table/), [drag and drop feature](/blog/2021-07-26-create-sql-tables-on-the-fly-with-zero-clicks/), [phoenix table support](/blog/2021-08-17-create-phoenix-tables-in-just-2-steps/), and [create table from excel file](/blog/2021-11-15-create-sql-tables-from-execl-files/) in Hue importer. With the help of Hue importer you can create tables in the above dialects easily just you need to load the data and submit. In this addition we are starting, supporting the iceberg table creation in Hive and Impala dialects.
+
+[Apache Iceberg](https://iceberg.apache.org/) is a high-performance format for huge analytic tables. Iceberg brings the reliability and simplicity of SQL tables to big data, while making it possible for engines like Spark, Trino, Flink, Presto, Hive and Impala to safely work with the same tables, at the same time.
+
+Create Iceberg table in just 2 steps.
+- Choose a file
```

Comment:
Use numbers instead of bullets if we are talking about sequential steps:
1. Go to the Importer and choose a file
2. Load the data and choose iceberg table option

Reply:
+1 -- always use the numbered list for sequential steps and the bulleted list for unordered items.

1. Go to the Hue importer and select a file from which you want to create a table.
2. Select the dialect from ... <specify the location on the UI>.
3. Select the Iceberg table option.
4. Configure the column types and tweak other parameters as needed.
5. Click Create.